test/blackbox/cloud-only.bats

```diff
@@ -1,3 +1,7 @@
+# Note: Intended to be run as "make run-blackbox-tests" or "make run-blackbox-cloud-ci"
```

Comment:
should we rename this file to cloud_only.bats for consistency ?

Reply:
it's renamed in 2 other prs